src/components/Layout/Footer.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import 'jest-pa11y/build/extendExpect';
+
+import { mount  } from 'enzyme';
+import { configurePa11y } from 'jest-pa11y';
+import { recorders } from 'jest-style-transformer-utils';
+import * as React from 'react';
+
+import Footer from './Footer';
+
+xdescribe('Footer', () => {
```

Comment:
How come this test is disabled?

Reply:
@kevcenteno Enabled the tests, along with adding an a11y specific Jest config so that unit tests that don't depend on Puppeteer aren't allocated a Puppeteer environment.